tests/e2e/raycluster_sdk_upgrade_test.py

```diff
@@ -0,0 +1,172 @@
+import requests
+from time import sleep
+
+from torchx.specs.api import AppState, is_terminal
+
+from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication
+from codeflare_sdk.job.jobs import DDPJobDefinition
+
+import pytest
+
+from support import *
+from codeflare_sdk.cluster.cluster import get_cluster
+
+namespace = "test-ns-rayupgrade"
+
+
+# Creates a Ray cluster
+@pytest.mark.openshift
```

Comment:
remove this marking as this test wont be run as part of e2e

Reply:
I was facing an error while importing support functions in separate upgrade folder as the codeflare-sdk init file wasn't there to be used as a package
```
from tests.e2e.support import *
E   ModuleNotFoundError: No module named 'tests'
```

Adding empty __init__ files in tests and upgrade folder resolved the import error !!
Is this approach seems feasible?